### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by haschek**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,7 +13,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 
 ### Changed
 
+-   GUI elements library need node 18 or an higher version because of necessary dependencies
```
Comment (andreas-schultz) [inline CHANGELOG.md:16]: Either change to `libraries` or `needs`. `an` -> `a`

File: README.md [hunk 2]
```diff
@@ -47,6 +47,25 @@ All [configuration variables](https://github.com/eccenca/gui-elements/blob/devel
 
 ## Development
 
+### Branch management
+
+We have 4 types of major branches representing the current state:
+
+-   `main`: contains the latest official release, only release branches will be merged in this branch
```
Comment (andreas-schultz) [inline README.md:54]: `in` -> `into`
Also, what are release branches? They do not seem to be described in the rest of the README even though the `release/` prefix is configured as allowed prefix.

File: README.md [hunk 3]
```diff
@@ -47,6 +47,25 @@ All [configuration variables](https://github.com/eccenca/gui-elements/blob/devel
...
+-   `develop`: contains the latest state of development, all feature and bugfix branches and here via PRs
```
Comment (andreas-schultz) [inline README.md:55]: Something seems to be missing in  `all feature and bugfix branches and here via PRs`

File: README.md [hunk 4]
```diff
@@ -47,6 +47,25 @@ All [configuration variables](https://github.com/eccenca/gui-elements/blob/devel
...
+-   `next`: development tree for an upcoming new major version, it will be merged into `develop` at some point
+-   `legacy`: development tree for the predecessor of the current major version, only bugfixes are going to merged in
```
Comment (andreas-schultz) [inline README.md:57]: `to merged` -> `to be merged`

File: src/components/AutoSuggestion/AutoSuggestionList.tsx [hunk 5]
```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import computeScrollIntoView from "compute-scroll-into-view";
+import { compute } from "compute-scroll-into-view";
```
Comment (andreas-schultz) [inline src/components/AutoSuggestion/AutoSuggestionList.tsx:2]: Naming the function just `compute` is too generic when used somewhere in the code.

Comment (haschek) [reply src/components/AutoSuggestion/AutoSuggestionList.tsx:2]: They changed the exports. This is the way to use it, see https://www.npmjs.com/package/compute-scroll-into-view?activeTab=readme#usage

**Review by andreas-schultz — APPROVED** [review]

**Pull request merged**